pulp_rpm/app/models/repository.py

```diff
@@ -40,6 +40,14 @@ class RpmRepository(Repository):
 
         sub_repo (Boolean):
             Whether is sub_repo or not
+        revision (Integer):
+            The revision number
+        remote_id (Integer):
+            The id of the remote
+        pulp_last_updated (DateTime):
```

Comment:
I assume this is remote_last_updated from below and not pulp_last_updated.

Reply:
good catch!